inotify.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procfs
+
+import (
+	"bufio"
+	"os"
+	"regexp"
+	"strings"
+)
+
```

Comment:
I think it would be good to create an additional struct here called `FDInfo` which contains pos, flags, and mnt_id since those fields appear to be consistently available.  The `FDInfo` struct could contain a slice of `InotifyStat` containing the detailed information for inotify file descriptors.
And to be more consistent with the rest of the API, the file should be called `proc_fdinfo.go`.